src/styles/main.scss

```diff
@@ -1,3 +1,79 @@
+$paddingCalendar: 10px;
+$gapFeature: 1px;
+$sideOfSquare: 100px;
+$widthOfCalendar: (
+  7 * $sideOfSquare) + (6 * $gapFeature) + (2 * $paddingCalendar
+);
+$marginFromLeftSide: (
+  "mon": 0,
+  "tue": 1,
+  "wed": 2,
+  "thu": 3,
+  "fri": 4,
+  "sat": 5,
+  "sun": 6
+);
+
 body {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.centered {
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: $widthOfCalendar;
+  padding: 10px;
+
+  @for $item from 1 through 31 {
+    &__day:nth-child(#{$item}) {
+      position: relative;
+      bottom: 0;
+      display: none;
+      transition: all 0.5s;
+      transition-timing-function: cubic-bezier(0, 0.48, 0.31, 0.93);
+      box-sizing: border-box;
+      width: $sideOfSquare;
+      height: $sideOfSquare;
+      border: 1px solid black;
+      background-color: #eee;
+      font-size: 30px;
```

Comment:
all these properties are the same for every item, move them out of `for` loop, to avoid repeating them. Just define them for `&__day` as usual